vm-setup/roles/common/defaults/main.yml

```diff
@@ -52,8 +52,8 @@ networks:
     address_v4: "{{ baremetal_network_cidr_v4|nthhost(1)|default('', true) }}"
     netmask_v4: "{{ baremetal_network_cidr_v4|ipaddr('netmask') }}"
     dhcp_range_v4:
-      - "{{ baremetal_network_cidr_v4|nthhost(20) }}"
-      - "{{ baremetal_network_cidr_v4|nthhost(60) }}"
+      - "{{ BAREMETALV4_POOL_RANGE_START }}"
+      - "{{ BAREMETALV4_POOL_RANGE_END }}"
```

Comment:
this is the metal3ippool range, it will end up in conflict if you add it to the virsh dhcp as it might be allocated statically by CAPM3 and dynamically by the dhcp

Reply:
ok got it.